src/main/java/address/controller/PersonCardController.java

```diff
@@ -126,17 +127,14 @@ protected String computeValue() {
      * Involves making an internet connection with the image hosting server.
      */
     private void setProfileImage() {
-        Optional<String> profileImageUrl = person.getGithubProfilePicUrl();
-        if (profileImageUrl.isPresent()){
-            new Thread(() -> {
-                Image image = new Image(profileImageUrl.get());
+        new Thread(() -> {
+                Image image = new Image(person.githubProfilePicUrl());
```

Comment:
What happens if the person has no Github URL? Won't this fail, as opposed to the original implementation? It could be a malformed URL, e.g. `github.com/<empty>.png`.